src/lib/utils.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+const autoGenerateProductPayload = () => {
+  const tags = {
+    'og:title': 'product_name',
+    'product:price:amount': 'price',
+    'product:price:currency': 'currency',
+    'og:description': 'description',
+    'og:url': 'url'
+  };
+
+  const attributes = Object.entries(tags).map(([ k, v ]) => {
```

Comment:
Variable names **k** and **v** is not self documented. It looks like it stands from **key** and **value**, but it isn't explain business value

Reply:
ok, I've made it more clearer.